tools/abel/README.md

```diff
@@ -0,0 +1,75 @@
+# ABEL
+#### Automated Building Entity Loader
+
+## Overview
+ABEL is a tool allowing system integrators to easily and efficiently create and modify [Building Configuration files](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md).
+
+#### UDMI
+ABEL does not yet support [UDMI Site Model](https://github.com/faucetsdn/udmi/blob/master/docs/specs/site_model.md).
+
+## Setup and installation
+Similar to the [GUID Generator](https://github.com/google/digitalbuildings/tree/master/tools/guid_generator), ABEL's setup process depends on the instance validator. It is best to run the global setup script to setup the entire tools library which ABEL depends on
```

Comment:
some users are not familiar with the toolkit and they dont have to since ABEL Will run the toolkit for them.
therefore this can be removed.